Lib/test/test_unicode.py

```diff
@@ -2830,6 +2830,28 @@ def test_asucs4(self):
             self.assertEqual(unicode_asucs4(s, len(s), True), s+'\0')
             self.assertEqual(unicode_asucs4(s, len(s), False), s+'\uffff')
 
+    # Test PyUnicode_GetUTF8Buffer()
+    @support.cpython_only
+    def test_getutf8buffer(self):
+        from _testcapi import unicode_getutf8buffer, test_unicode_getutf8buffer
+
+        # Run tests wrtten in C.  Raise an error when test failed.
+        test_unicode_getutf8buffer()
```

Comment:
Hum. test_capi executes all test_capi attributes with a name starting with "test_". Please check if it's executed. If yes, just rename the method. For example, rename it to unicode_test_getutf8buffer()).

Reply:
You are right.  I confirmed the test_unicode_getutf8buffer is called from test_capi.

I just removed calling test_unicode_getutf8buffer here because there is `test_unicode_compare_with_ascii` already.
Let's run the test from test_capi.